vars/restoreCache.groovy

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/groovy
+
+def call(String file) {
+    try {
+        sh """
```

Comment:
If we used `'''` instead of `"""` here we would not have have to do all the escaping below.

Reply:
I'm using `${file}` inside.